vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -791,6 +766,12 @@ public boolean onPrepareOptionsMenu(Menu menu) {
         if (CommonActivityUtils.shouldRestartApp(this)) {
             return false;
         }
+        //no need to setup the searchview again if its already done
+        if(!getResources().getBoolean(R.bool.enable_riot_search_view) && mSearchView == null) {
```

Comment:
Was expecting this to be:
    if(getResources().getBoolean(R.bool.enable_riot_search_view) && mSearchView == null) {
e.g. no not considering the boolean is enable riot search view

Reply:
I think this is right. I want to access into the block if enable_riot_search_view = false.

Reply:
This is confusing, please add a code comment explaining what is going on ... as you mentioned in another code ... there are two search views ?

Reply:
The way Riot used the searchview, it is meant to be collapsed all the time. However, Lance decided to have a search view which is collapsed and the app name "Lingo" can be seen.